pkgs/servers/consul/default.nix

```diff
@@ -1,22 +1,26 @@
-{ stdenv, buildGoPackage, consul-ui, fetchFromGitHub }:
+{ stdenv, buildGoPackage, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGo110Package rec {
```

Comment:
IS this a typo?

Reply:
if not, I don't understand where this identifier comes from :)

Reply:
Typo from my forward-port, fixed